fairscale/optim/oss.py

```diff
@@ -52,6 +54,10 @@ class OSS(Optimizer):
             torch.distributed group (default: group.WORLD)
         broadcast_buffer_size (int):
             (deprecated) used to cap the size of the broadcast buffers, not being used anymore.
+        broadcast_fp16 (bool):
+            Compress the model shards in fp16 before sharing them in between ranks.
```

Comment:
model shards or optimizer state shards?

Reply:
model shards, really, not like in FDSP since each rank has all of them, but these are real model shards